packages/devextreme/js/__internal/ui/chat/chat_message_box.ts

```diff
@@ -40,8 +60,41 @@ class MessageBox extends Widget<any> {
     this._button = this._createComponent($button, Button, {
       icon: 'send',
       stylingMode: 'text',
+      onClick: (e): void => {
+        this._buttonClickHandler(e);
+      },
     });
   }
+
+  _createMessageSendAction(): void {
+    this._messageSendAction = this._createActionByOption(
+      'onMessageSend',
+      { excludeValidators: ['disabled', 'readOnly'] },
+    );
+  }
+
+  _buttonClickHandler(e: ClickEvent): void {
```

Comment:
```suggestion
  _sendHandler(e: ClickEvent): void {
```

Reply:
I would prefer to name it more common because as we discussed earlier it might be used in case when keyboard is using

Reply:
[cd52ce11e169517258f5b51a5b14ac75315205da]